frontend/www/lib/ionic/scss/_loading.scss

```diff
@@ -0,0 +1,51 @@
+
+/**
+ * Loading
+ * --------------------------------------------------
+ */
+
+.loading-container {
+  position: absolute;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered bottom, left, opacity, position, right, top, visibility, z-index